week1/Q1_test.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import Q1
+
+
+class PermutationTest(unittest.TestCase):
+
+	def testExampleCases(self):
```

Comment:
Thanks for review! I had a very important deadline, had no time to look at anything else, sorry.
Actually, I can't think of anything except empty strings. What else could've been checked?